test/oats/kafka/yaml/oats_go_kafka.yaml

```diff
@@ -15,23 +15,23 @@ expected:
             messaging.destination.name: important
             messaging.operation.type: process
             messaging.system: kafka
-            #server.port: "9092"
```

Comment:
do we want to keep all these commented ports? what's the change here?

Reply:
I missed enabling those, I'll enable them.

Reply:
We want to check the ports, we had a bug in the way we ordered the events and the destination port was an ephemeral port. The reason I couldn't add the tests before was because OATS couldn't compare integers. Now it does after I made fixes to the framework.